source/_integrations/group.markdown

```diff
@@ -36,7 +38,7 @@ The group integration lets you combine multiple entities into a single entity. E
 This can be useful for cases where you want to control, for example, the
 multiple bulbs in a light fixture as a single light in Home Assistant.
 
-Home Assistant can group multiple binary sensors, covers, fans, lights, locks, media players, switches as a single entity, with the option of hiding the individual member entities.
+Home Assistant can group multiple binary sensors, climate devices, covers, fans, lights, locks, media players, switches as a single entity, with the option of hiding the individual member entities.
```

Comment:
These are not devices, but entities.

```suggestion
Home Assistant can group multiple binary sensors, climate, covers, fans, lights, locks, media players, switches as a single entity, with the option of hiding the individual member entities.
```

Reply:
In this case, I meant an actual physical device, not in the "home-assistant" meaning. But I can see that this might be confusing. 
Then I would propose:
 
```suggestion
Home Assistant can group multiple binary sensors, climate entities, covers, fans, lights, locks, media players, switches as a single entity, with the option of hiding the individual member entities.
```